gcn/notices/core/GeoLocBase.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/core/GeoLocBase.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Geolocation (Generic)",
+  "description": "A generic schema for position and velocity in geocentric coordinates.",
+  "$comment": "Providers shouldn't $ref this schema directly, but GeoLoc.schema.json instead.",
+  "type": "object",
+  "properties": {
+    "latitude": { "type": "number", "description": "Geocentric lat-lon" },
+    "longitude": { "type": "number", "description": "Geocentric lat-lon" },
+    "radius": {
+      "type": "number",
+      "description": "Distance from the center of the Earth."
+    },
+    "x": { "type": "number", "description": "Cartesian geocentric" },
+    "y": { "type": "number", "description": "Cartesian geocentric" },
+    "z": { "type": "number", "description": "Cartesian geocentric" },
+    "vx": { "type": "number", "description": "Cartesian geocentric velocity" },
+    "vy": { "type": "number", "description": "Cartesian geocentric velocity" },
+    "vz": { "type": "number", "description": "Cartesian geocentric velocity" },
```

Comment:
What is the use case for including the state vector in alerts? For near-term orbit determination? I think that there are better sources for ephemerides than GCN.

Reply:
@lpsinger: You are correct on the firs part. About the ephemerides: How many sources do you have in mind and are all data readily computer readable? What is the latency?

Reply:
No other instrument provides ephemerides in GCN. What use do you see for follow-up observers in having the state vectors?

Reply:
I obviously misunderstood that it needs to be in the schema when we were were talking about it at the meeting.

Reply:
I think less is more right now. We are aiming to capture the minimum that all missions and experiments need, not account for everything that any mission may need.